fast_api/routes/data_slices.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional
+
+from fastapi import APIRouter, Depends, Request
+from fast_api.routes.client_response import pack_json_result, wrap_content_for_frontend
+from submodules.model.util import sql_alchemy_to_dict
+from typing import List
+from controller.data_slice import manager
+from controller.record import manager as record_manager
+from controller.auth import manager as auth_manager
+from util import notification
+import json
+
+
+router = APIRouter()
+
+
+@router.get(
+    "/{project_id}", dependencies=[Depends(auth_manager.check_project_access_dep)]
+)
```

Comment:
I think it makes to add /projectId/data-slices for getting all dataSlices, since this request is used in many pages and it would be much easier to find it with 'name' than with id.(not crutual for all requests but i think for this one makes sense).
- [ ] resolved
- [ ] ignored

Reply:
renamed to get-data-slices for clarity and searchability